### ISSUE TITLE
Stuck on getting TAS data, bot wont run

### ISSUE THREAD

**Issue by manfsdagawd** [natewong1313/bird-bot#75]
Runnig through a proxy server, moved computers, even changed locations. Bot still stuck on getting TAS data. Also, bot worked on my desktop mac, then updated to mac os big sur 11.0.1, bot wont run, python immediately displays "application not running" whenever I try to launch bot. Might be running through proxy incorrectly on my macbook. Desktop mac had the same TAS issue prior to update. Now after update it wont run at all. Not sure if this bot still gets updates/work done but it's a very user friendly interface and I like it a lot. Haven't successfully purchased anything yet but I hope to soon. Also hope to find help here asap. Thank you.

**Comment by jleezy8** [comment]
The projects been over almost 8 month ago.

**Comment by jleezy8** [comment]
Projects been dead for 8 months. You can check in the post right before yours called Bot is dead..

**Comment by rjanson** [comment]
I was having the same issue with the Best Buy module. I updated the user-agent string in bestbuy.py to be "Mozilla/5.0 (Macintosh; Intel Mac OS X 11_0_0) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/86.0.4240.198 Safari/537.36" (in 16 places) and it started working.

I was not having any issues with the Walmart module so no changes needed.

**Comment by vCloud2K** [comment]
just like jleezy8 said "the project has been dead" please check out the post "The bot is dead" to view the new bot the community has made(It's open source just like Bird Bot and being updated)

**Comment by topfiy** [comment]
Where is the new bot?

**Comment by fbarnett6** [comment]
rjanson has the right idea. I just checked my mac osx version and chrome version, then changed every instance inside bestbuy.py. Sure enough it did the trick for me.

**Comment by jleezy8** [comment]
a lot of kids from snkr twitter on here trying to get a free bot lol

**Comment by topfiy** [comment]
Naw I don't want a free bot. Resales isn't my thing and I don't judge but scalping doesn't sit morally right with me. I just decided to try and get into coding and was like let me try by making a bot that can just buy and check out for me. I'm trying to get it to buy the game of life for me..... I suck at this and it's hard to learn alone with no teacher. So I was trying to maybe see how so of y'all ran the code. > a lot of kids from snkr twitter on here trying to get a free bot lol

**Comment by jleezy8** [comment]
IM sorry i wasn't necessarily directing that at you, im just saying there's been an influx of commenters who just joined this month. But yeah i get you. You have to compile it yourself.

**Comment by topfiy** [comment]
Yes yes I totally understand that. But I'm currently working on it. But see I see you all who are actually coders and programs are struggling so it just showed me that this might be too much out of my skill cap.

**Comment by jleezy8** [comment]
join the new server here: https://discord.gg/gJ2ABW2F

**Comment by Antareann** [comment]
> https://discord.gg/gJ2ABW2F

Hi, can you send a new invite? That one expired already.

**Comment by jleezy8** [comment]
yeh there limited use, heres another:
https://discord.gg/j6NtK7KjY8
